docs/user_manual_how_to_use_1_operator_mode_additional_functionalities.rst

```diff
@@ -4,35 +4,34 @@
 Additional Functionalities
 **************************
 
-Create a WOA or RTOFS profile
------------------------------
 
-.. index:: WOA, synthetic profile
-.. index:: RTOFS, synthetic profile
+Create profile from database
+============================
+
+.. index:: profile, synthetic profile
+
+The package makes it possible to create a single profile from a number of database options. This can be done by
+selecting one of the options listed under *Retrieve from* in the *Input data* dialog (:numref:`editor_tab12`). Database
+specific instructions for creating a profile are detailed in the sections below.
 
-It is possible to upload a single WOA or RTOFS profile to *SIS*.
-This can be done by first selecting WOA09 atlas or RTOFS atlas under the *Input data* button in the *Editor* toolbar
-(:numref:`editor_tab12`).
 
 .. _editor_tab12:
 .. figure:: ./_static/editor_tab12.png
-    :width: 360px
+    :width: 600px
     :align: center
     :alt: data storage
     :figclass: align-center
 
     The *Input data* button in the *Editor* toolbar.
 
-This will trigger a series of question dialogs about timestamp and position to apply a spatio-temporal search.
-The user can decide to use the SIS timestamp/position input (when available) or manually set these inputs.
-After, a surface sound speed can be applied, and finally the resulting cast can be sent as described in :ref:`data_transmission`.
-
-The new cast will be given the filename YYYYMMDD_HHMMSS_WOA or YYYYMMDD_HHMMSS_RTOFS with the date/time
-in the filename based on the query time of the cast.
-
+Project Database
+----------------
+The *Input data* dialog can be used to recall a profile from the project database. Clicking the *Project DB* button will
+open a dialog with a drop-down menu containing all the profiles in the current project database. After selecting a
+profile, it can be edited, and finally the resulting cast can be sent as described in :ref:`data_transmission`.
```

Comment:
I saw that exporting the data after editing it did not add a new entry to the database. Should I add a note about this?

Reply:
It does not add a new entry (but overwrite the existing one) since the primary key is the same: location + timestamp
This is done on purpose to avoid data duplication.